resources/helm-broker/README.md

```diff
@@ -0,0 +1,7 @@
+# Helm Broker
+
+## Overview
+
+Helm Broker provides bundles in the [Service Catalog](../service-catalog/README.md). A bundle is an abstraction layer over a Helm chart which enables you to provide more information about the Helm chart. For example, a bundle can provide plan definitions or binding details. Service Catalog requires this information. Bundles are services available in Service Catalog.
+
+Helm Broker implements the [Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md). For more information about the Service Brokers, see the [Service Brokers overview](../../../../docs/service-brokers/docs/001-overview-service-brokers.md) documentation. Find the details about the Helm Broker in the [docs](../../docs/service-brokers/docs) repository.
```

Comment:
Could you please fix the broken Service Brokers overview link? :(

Reply:
Fixed